test/lib/test/config_test.exs

```diff
@@ -0,0 +1,17 @@
+# Copyright(c) 2015-2020 ACCESS CO., LTD. All rights reserved.
+
+Antikythera.Test.Config.init()
```

Comment:
If you have no intention, I think this line can be removed.

```suggestion
```

Reply:
Removed at https://github.com/access-company/antikythera/pull/119/commits/6605e60b5bf1b6cd4efd0a15b049289d485500a2